formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java

```diff
@@ -43,7 +44,10 @@ public final class JsonFormat implements EventFormat {
      * Content type associated with the JSON event format
      */
     public static final String CONTENT_TYPE = "application/cloudevents+json";
-
+    /**
+     * Suppoted Content type
```

Comment:
I am not sure if it's good to call it "Supported Content type".
Other content type should work too, they will just be exposed as `byte[]` not objects IIRC.

Reply:
Understood, any suggestion for the naming?

Reply:
How about something along the lines of "_JSON Data Content Type Discriminator"_ - long-winded I known ;-)

Reply:
@JemDay  Done!